atomics/include/desul/atomics/Fetch_Op_OpenMP.hpp

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright (c) 2019, Lawrence Livermore National Security, LLC
+and DESUL project contributors. See the COPYRIGHT file for details.
+Source: https://github.com/desul/desul
+
+SPDX-License-Identifier: (BSD-3-Clause)
+*/
+#ifndef DESUL_ATOMICS_FETCH_OP_OPENMP_HPP_
+#define DESUL_ATOMICS_FETCH_OP_OPENMP_HPP_
+
+#include <desul/atomics/Common.hpp>
+#include <desul/atomics/openmp/OpenMP_40.hpp>
+
+#if 0
```

Comment:
Is this all waiting on 5.1 support or what's the story? Can we convert the if 0 to a check on `_OPENMP` or similar perhaps?

Reply:
To be honest I don't remember.  @crtrott any idea?

Reply:
I don't see where this would be coming from there is no such commented out code in the original is there?

Reply:
OK now I remember.  It was my looking into refactoring https://github.com/desul/desul/blob/497f6032effb2867e87428cbd964bcdc4183ca91/atomics/include/desul/atomics/openmp/OpenMP_40.hpp#L62-L96
and then realizing they are enough moving parts and I should probably do that as a follow up PR.
I added a FIXME.  Let me know if you prefer me to remove.

Reply:
The fixme is fine, so long as we have breadcrumbs to come back.